gateway/src/apicast/policy/nginx_metrics/nginx_metrics.lua

```diff
@@ -73,6 +73,12 @@ local http_connections_metric =  prometheus('gauge', 'nginx_http_connections', '
 local shdict_capacity_metric = prometheus('gauge', 'openresty_shdict_capacity', 'OpenResty shared dictionary capacity', {'dict'})
 local shdict_free_space_metric = prometheus('gauge', 'openresty_shdict_free_space', 'OpenResty shared dictionary free space', {'dict'})
 
+local response_times = prometheus(
+  'histogram',
+  'response_time_seconds',
```

Comment:
I think we will have more "response time" metrics, so we should name them well. In this case I'd mention it is "total" response time, because in the future we will have "upstream" response time metric for sure.

Reply:
Fixed :+1: 
I also adapted the name of the metric that tracks upstream response times to follow the same format. It was introduced after the latest release, so it's not a breaking change.